src/actions/pause.js

```diff
@@ -91,7 +98,9 @@ export function paused(pauseInfo: Pause) {
       dispatch(removeBreakpoint(hiddenBreakpointLocation));
     }
 
-    dispatch(evaluateExpressions(frame.id));
+    if (!hasWatchExpressionErrored(getState())) {
```

Comment:
Could you add a comment pointing to the original issue so someone reading this can have some context ?